src/components/Disclaimer/Disclaimer.tsx

```diff
@@ -22,9 +22,11 @@ const Disclaimer = ({ metricName }: { metricName: number }) => {
         {metricName === Metric.CASE_DENSITY && (
           <Fragment>
             <span>
-              Our risk levels for daily new cases are based on the “Key Metrics
-              for Covid Suppression” by Harvard Global Health Institute and
-              others.
+              Our risk levels for daily new cases are based on the{' '}
+              <a href="https://ethics.harvard.edu/files/center-for-ethics/files/key_metrics_and_indicators_v4.pdf">
+                “Key Metrics for Covid Suppression”
+              </a>{' '}
```

Comment:
Other external links open in a new tab `target="_blank" rel="noopener noreferrer"`, not sure if we want that in this case.

Reply:
Oops! Good call.